src/main/java/com/giantshoulder/pageobject/MainPage.java

```diff
@@ -0,0 +1,131 @@
+package com.giantshoulder.pageobject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+
+import static com.giantshoulder.util.WebElementHandler.*;
+
+/**
+ * @author Gabriella_Havadtoi
+ */
+public class MainPage extends BasePage {
+
+    public MainPage(ChromeDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(id = "element3421730-content")
+    WebElement subscribeButton;
```

Comment:
All these WebElements could be private I think.

Reply:
Corrected!